ticdc: add upgrading section (#3906)
pingcap/docs: ticdc/manage-ticdc.md

- `cert`: The path of the certificate file used by TiCDC, in the PEM format (optional).
- `key`: The path of the certificate key file used by TiCDC, in the PEM format (optional).

## Upgrade TiCDC using TiUP

> This section introduces how to upgrade the TiCDC-enabled cluster using TiUP. In the following example, assume that you need to upgrade TiCDC and the cluster to v4.0.6.

Reviewer: Waiting on https://github.com/pingcap/docs-cn/pull/4503#pullrequestreview-488344750

Author: ```suggestion
This section introduces how to upgrade the TiCDC cluster using TiUP. In the following example, assume that you need to upgrade TiCDC and the entire TiDB cluster to v4.0.6.
```

Reviewer: LGTM

{{< copyable "shell-regular" >}}

```shell
tiup update --self && \
tiup update --all && \
tiup cluster upgrade <cluster-name> v4.0.6
```

### Notes for upgrade

> * In TiCDC v4.0.2, changes were made to the `changefeed` configuration. See [Compatibility notes for the configuration file](/production-deployment-using-tiup.md#step-3-edit-the-initialization-configuration-file) for details.

Reviewer: ```suggestion
* The `changefeed` configuration has changed in TiCDC v4.0.2. See [Compatibility notes for the configuration file](/production-deployment-using-tiup.md#step-3-edit-the-initialization-configuration-file) for details.
```
